app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+require'pry'
```

Comment:
You don't have to require 'pry' here. and it should be deleted for review anyway.